home/management/commands/create_default_site.py

```diff
@@ -1,16 +1,33 @@
 from django.conf import settings
 from django.contrib.auth import get_user_model
-from django.contrib.sites.models import Site
+from django.contrib.sites.models import Site as DjangoSite
 from django.core.management.base import BaseCommand
+from wagtail.core.models import Site as WagtailSite
+
+from home.models import HomePage
 
 User = get_user_model()
 
 
 class Command(BaseCommand):
 
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--domain',
+            help='The domain on which the website is deployed'
+        )
```

Comment:
Can we add a port parameter which affects the WagtailSite object? (It seems that the DjangoSite object does not have a port associated with it)